app/controllers/ContactHmrcController.scala

```diff
@@ -174,11 +171,8 @@ class ContactHmrcController @Inject() (
   }
 
   private def doThanks(implicit request: Request[AnyRef]) = {
-    val result = request.session.get("ticketId").fold(BadRequest("Invalid data")) { ticketId =>
-      val view =
-        if (appConfig.enablePlayFrontendContactHmrcForm) playFrontendContactConfirmationPage()
-        else assetsFrontendContactConfirmationPage(ticketId)
-      Ok(view)
+    val result = request.session.get("ticketId").fold(BadRequest("Invalid data")) { _ =>
```

Comment:
Template no longer needs `ticketId` parameter, and this whole check about the `ticketId` in the session should be removed.